.github/workflows/run-tests-2.yml

```diff
@@ -0,0 +1,33 @@
+name: 'Run tests 2'
```

Comment:
Let's combine test-1 and test-2. It was split to test the theory that they can run successfully separately.